test/configCases/cache-filesystem/multicompiler-mode-cache-warning/warnings.js

```diff
@@ -0,0 +1,4 @@
+module.exports = [
+	/Please set unique "cache\.name" option/,
+	/Compiler with name "3rd compiler" doesn't use unique cache name/
```

Comment:
I am afraid it can be spammy, I saw a lot of `type: "filesystem"` without cache name and multi compiler mode, maybe better output a warning when we have explicit identical names?

Reply:
ok, lets do this way

Reply:
done